### PR TITLE
debug : footer-line break issue

### PR DESCRIPTION
**Pull request by wooseok123**
### changes :
`margin-left` of `dark-mode-toggle` : `1em` to `0.15em`
`font-size` of `nav` : `16px` to `15px`

 I've changed `margin-left` value of `dark-mode-toggle` to `0.15em` and `font-size` of anchor tag in `#footer nav` to `15px` so that the letter 'light' and 'dark' can be in the same line until `819px`.

The layout changes when the width is under `818px`.  So i think it would be better if there's no line break until `819px`.
So it works on my code!

### Before 
(over `860px`)
image
image

### After 
(over `860px`, it maintains the same line until `819px`)
image
image

### CODE REVIEW

**Review by Nitinkumar2905 — COMMENTED** [review]
This change in font size really solved the line break on the theme change.

File: src/_css/main.css [hunk 1]
```diff
@@ -224,6 +224,10 @@ hr {
   align-items: flex-end;
 }
 
+#footer nav {
+  font-size : 15px;
```
Comment (camillobruni) [inline src/_css/main.css:228]: I'd prefer not adding a new font-size.
How about removing the dots `.` separators between the nav links and add `margin-right: 0.5em` for `footer nav a`?

Comment (wooseok123) [reply src/_css/main.css]: I agree with your opinion.
I've just added the modified version. (removing dots and adding `margin-right` to them, and things i had to do to align them in the right position.)

The line-break still remains in the mobile version, but i think it's not a big things.

**Comment by wooseok123** [comment]
@camillobruni If there aren’t any major issues, how about merging it?

File: src/_css/main.css (hunk 1 above)
Comment (camillobruni) [reply src/_css/main.css]: adding a .prettierrc sounds good to me

**Pull request merged**